enterprise-suite/Makefile

```diff
@@ -105,20 +105,20 @@ install-helm:  ## Install required helm components into cluster
 	-kubectl create namespace $(TILLER_NAMESPACE)
 	-kubectl create serviceaccount --namespace $(TILLER_NAMESPACE) tiller
 	-kubectl create clusterrolebinding $(TILLER_NAMESPACE):tiller --clusterrole=cluster-admin --serviceaccount=$(TILLER_NAMESPACE):tiller
-	-helm init --wait --service-account tiller --tiller-namespace=$(TILLER_NAMESPACE)
+	-helm init --wait --service-account tiller --upgrade --tiller-namespace=$(TILLER_NAMESPACE)
 
 # called from shell script for travis job:
 install-local: $(HELM_CHARTS_DIR)/docs/$(RELEASE).tgz install-helm  ## Install local tarball
 	# We set a podUID here for test purposes to ensure everything works as non-root.
-	-ES_LOCAL_CHART=$(HELM_CHARTS_DIR)/docs/$(RELEASE).tgz ES_FORCE_INSTALL=true \
```

Comment:
Interesting.  Why the change?  (Just curious.)

Reply:
I was just getting frustrated with the 60s+ redeploy / dev cycle :). Removing force install and using upgrade works fine 99% of the time and only takes seconds. Maybe it's controversial? Not sure. I mentioned this in slack and no one replied, so I assume it's okay...